kroxylicious-docs/docs/authorization-guide/index.adoc

```diff
@@ -0,0 +1,36 @@
+:experimental:
+include::_assets/attributes.adoc[]
+
+:context: authorization
+:guide: authorization
+
+[id="using-book-{context}"]
+= Authorization Guide
+
+include::_modules/authorization/con-about-{guide}-guide.adoc[leveloffset=+1]
+
+[role="_abstract"]
+The Authorization filter gives the ability to add authorization checks into a Kafka system which will be enforced by the proxy.
+These authorization checks are performed in addition to any authorization checks made by the broker itself.
+For an action to be allowed, both the authorization filter and broker must decide that the action is allowed.
+
+In general, the Authorization filter will make access decisions in the same manner as Kafka itself.
+This means it will apply the same authorizer checks that Kafka enforces itself and generate error responses in the same way.
+Indeed, from the client's perspective, it will be impossible for it to distinguish between the proxy and kafka cluster itself.
```

Comment:
```suggestion
From the client's perspective, it will be impossible for it to distinguish between authorization enforced on the proxy and authorization enforced on the kafka cluster itself.
```